### PR TITLE
Cleanup unused hosting interfaces

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

File: src/vm/commtmemberinfomap.cpp [hunk 1]
```diff
@@ -1381,6 +1380,67 @@ void ComMTMemberInfoMap::AssignNewEnumMember(
     }
 } // void ComMTMemberInfoMap::AssignNewEnumMember()
 
+//*****************************************************************************
+// Signature utilities.
+//*****************************************************************************
+class MetaSigExport : public MetaSig
```
Comment (jkotas) [inline src/vm/commtmemberinfomap.cpp]: This is copy&paste from tlbexport.h

File: src/inc/utilcode.h [hunk 2]
```diff
@@ -4200,7 +4200,7 @@ void TrimWhiteSpace(__inout_ecount(*pcch)  LPCWSTR *pwsz, __inout LPDWORD pcch);
 HRESULT Utf2Quick(
     LPCUTF8     pStr,                   // The string to convert.
     CQuickArray<WCHAR> &rStr,           // The QuickArray<WCHAR> to convert it into.
-    int         iCurLen);               // Inital characters in the array to leave (default 0).
```
Comment (AaronRobinsonMSFT) [inline src/inc/utilcode.h]: ~~Inital~~ -> Initial

**Comment by jkotas** [comment]
The interfaces that I am deleting are either SQL hosting-specific, WindowsPhone-specific or not used at all.

**Comment by jkotas** [comment]
@dotnet-bot test Ubuntu x64 Checked CoreFX Tests  please

**Pull request merged**